index.rst

```diff
@@ -3,11 +3,6 @@
 Symfony Documentation
 =====================
 
-.. toctree::
-    :hidden:
-
-    changelog
```

Comment:
it isn't now, as symfony.com builds use another `conf.py` setting. Otherwise, we can always use http://stackoverflow.com/questions/13529985/how-to-set-sphinxs-exclude-patterns-from-the-command-line